docs/sources/reference/components/pyroscope/pyroscope.ebpf.md

```diff
@@ -43,26 +47,35 @@ The component configures and starts a new eBPF profiling job to collect performa
+| `pid_cache_size`          | `int`                    | Deprecated (no-op), previously controlled the size of the PID -> proc symbols table LRU cache                                    | 32      | no       |
 | `pid_map_size`            | `int`                    | The size of eBPF PID map                                                                                                         | 2048    | no       |
 | `python_enabled`          | `bool`                   | A flag to enable/disable python profiling                                                                                        | true    | no       |
-| `same_file_cache_size`    | `int`                    | The size of the elf file -> symbols table LRU cache                                                                              | 8       | no       |
+| `perl_enabled`            | `bool`                   | A flag to enable/disable perl profiling                                                                                          | true    | no       |
+| `php_enabled`             | `bool`                   | A flag to enable/disable php profiling                                                                                           | true    | no       |
+| `hotspot_enabled`         | `bool`                   | A flag to enable/disable hotspot profiling                                                                                       | true    | no       |
+| `v8_enabled`              | `bool`                   | A flag to enable/disable v8 profiling                                                                                            | true    | no       |
+| `ruby_enabled`            | `bool`                   | A flag to enable/disable ruby profiling                                                                                          | true    | no       |
+| `dotnet_enabled`          | `bool`                   | A flag to enable/disable dotnet profiling                                                                                        | true    | no       |
+| `same_file_cache_size`    | `int`                    | Deprecated (no-op), previously controlled the size of the elf file -> symbols table LRU cache                                    | 8       | no       |
 | `sample_rate`             | `int`                    | How many times per second to collect profile samples                                                                             | 97      | no       |
-| `symbols_map_size`        | `int`                    | The size of eBPF symbols map                                                                                                     | 16384   | no       |
+| `symbols_map_size`        | `int`                    | Deprecated (no-op), previously controlled the size of eBPF symbols map                                                           | 16384   | no       |
 
```

Comment:
It would be good to also pass through `OTEL_PROFILING_AGENT_VERBOSE=1` as an argument

Reply:
On the one hand I like this idea, but on the other I want to introduce as little arguments as possible. It is currently possible to enable this flag through env variable. I'd prefer to keep it an undocumented debugging-facility feature. WDYT?